services/worker/src/worker/job_runners/config_size.py

```diff
@@ -0,0 +1,211 @@
+    dataset: str
+    config: str
+    split: str
+    num_bytes_parquet_files: int
+    num_bytes_memory: int
+    num_rows: int
+    num_columns: int
+
+
+class ConfigSizeContent(TypedDict):
+    config: ConfigSize
+    splits: list[SplitSize]
+
+
+class ConfigSizeResponse(TypedDict):
+    size: ConfigSizeContent
+
+
+class ConfigSizeJobRunnerError(JobRunnerError):
+    """Base class for exceptions in this module."""
+
+    def __init__(
+        self,
+        message: str,
+        status_code: HTTPStatus,
+        code: ConfigSizeJobRunnerErrorCode,
+        cause: Optional[BaseException] = None,
+        disclose_cause: bool = False,
+    ):
+        super().__init__(
+            message=message, status_code=status_code, code=code, cause=cause, disclose_cause=disclose_cause
+        )
+
+
+class PreviousStepStatusError(ConfigSizeJobRunnerError):
+    """Raised when the previous step gave an error. The job should not have been created."""
+
+    def __init__(self, message: str, cause: Optional[BaseException] = None):
+        super().__init__(message, HTTPStatus.INTERNAL_SERVER_ERROR, "PreviousStepStatusError", cause, False)
+
+
+class PreviousStepFormatError(ConfigSizeJobRunnerError):
+    """Raised when the content of the previous step has not the expected format."""
+
+    def __init__(self, message: str, cause: Optional[BaseException] = None):
+        super().__init__(message, HTTPStatus.INTERNAL_SERVER_ERROR, "PreviousStepFormatError", cause, False)
+
+
+class MissingInfoForConfigError(ConfigSizeJobRunnerError):
+    """Raised when the dataset info from the parquet export is missing the requested dataset configuration."""
+
+    def __init__(self, message: str, cause: Optional[BaseException] = None):
+        super().__init__(message, HTTPStatus.INTERNAL_SERVER_ERROR, "MissingInfoForConfigError", cause, False)
+
+
+def compute_config_size_response(dataset: str, config: str) -> ConfigSizeResponse:
+    """
+    Get the response of /sizes for one specific dataset and config on huggingface.co.
+    Args:
+        dataset (`str`):
+            A namespace (user or an organization) and a repo name separated
+            by a `/`.
+        config (`str`):
+            A configuration name.
+    Returns:
+        `ConfigSizeResponse`: An object with the size_response.
+    <Tip>
+    Raises the following errors:
+        - [`~job_runners.config_size.PreviousStepStatusError`]
+          If the previous step gave an error.
+        - [`~job_runners.config_size.PreviousStepFormatError`]
+            If the content of the previous step has not the expected format
+    </Tip>
+    """
+    logging.info(f"get size for dataset={dataset}, config={config}")
+
+    try:
+        response = get_response(kind="/parquet-and-dataset-info", dataset=dataset)
+    except DoesNotExist as e:
+        raise DatasetNotFoundError(
+            "No response found in previous step for this dataset: '/parquet-and-dataset-info' endpoint.", e
```

Comment:
```suggestion
            "No response found in previous step for this dataset: '/parquet-and-dataset-info'.", e
```

Reply:
this is also present in dataset_info.py and parquet.py, I guess I can do the changes in those as well while I'm at it